src/app/components/ThemeToggle.tsx

```
@@ -0,0 +1,71 @@
+"use client";
+
+import React from "react";
+import { IconButton, Tooltip, Box } from "@mui/material";
+import { Sun, Moon } from "lucide-react";
+import { useThemeMode } from "./ThemeRegistry";
+import { alpha } from "@mui/material/styles";
+
+export interface ThemeToggleProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)